packages/components/src/elevation/types.ts

```diff
@@ -35,18 +39,22 @@ export type Props = {
 	 */
 	offset?: number;
 	/**
-	 * Size of the shadow, based on the Style system's elevation system. The `value` determines the strength of the shadow, which sense of depth.
-	 * In the example below, `isInteractive` is activated to give a better sense of depth.
+	 * Size of the shadow, based on the Style system's elevation system. The
+	 * `value` determines the strength of the shadow, which sense of depth.
```

Comment:
I _think_ this is what's missing in this phrase?

```suggestion
	 * `value` determines the strength of the shadow, which creates the sense of depth.
```